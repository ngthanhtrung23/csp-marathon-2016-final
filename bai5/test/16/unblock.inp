.12223
.14..3
004.56
788856
7.9ABB
..9ACC
