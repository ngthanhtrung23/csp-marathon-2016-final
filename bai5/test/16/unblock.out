25
5 U 1
7 D 1
8 L 1
A U 3
B L 1
C L 1
6 D 2
3 D 1
2 R 1
4 U 1
0 R 1
8 R 3
7 U 4
0 L 1
9 U 2
B L 3
9 D 2
8 L 3
5 D 2
A D 2
0 R 3
4 D 1
2 L 1
3 U 1
0 R 1

